migration/db-id/mssql/migration.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+set -e
+
+get_db_id_migration_path () {
+  search="eventuateCommonVersion="
+  version_line="$(grep $search ./gradle.properties)"
+  version=${version_line#$search}
+  echo "${db_id_migration_repository}/${version}"
```

Comment:
This should be uppercase and optional, e.g. default to this repo.

Reply:
ok